### ISSUE TITLE
Misplaced figures in auto-generated proofs

### ISSUE THREAD

**Issue by dmey** [openjournals/joss#472]
I am having an issue in openjournals/joss-reviews#1113 with the auto-generated proof -- see [this](https://github.com/openjournals/joss-reviews/issues/1113#issuecomment-444672650) -- the figure ends up in the reference list instead of the summary section.

**Comment by danielskatz** [comment]
I don't think there's any option.  Based on where you have the figure reference in the text (near the end of the first page), the figure cannot go on the same page after it, so it's forced to the next page (which happens to be in the references.)

**Comment by dmey** [comment]
I am not sure what you mean -- the figure is added just after the 'Summary' heading

``` markdown
# Summary

![Relationships of common functions as implemented in PsychroLib. Bold arrows show the relationship between function involving a direct call while light arrow show the relationship between two or more. For a complete list of functions available in PsychroLib, see the README file in the project’s repository.](psychrolib-relationships.pdf){ width=20% }
```

see: https://github.com/psychrometrics/psychrolib/blob/7d58638002f2bf6046d745a131ceeecc895a84d4/paper/paper.md

Is there a way to force verbose output?

**Comment by arfon** [comment]
Figure placement in pandoc is a black art as far as I can tell.

I've just tried the following @dmey and it produced [this pdf](https://github.com/openjournals/joss/files/2652894/10.21105.joss.01113.pdf):

```
# Summary

![Relationships of common functions as implemented in PsychroLib. Bold arrows show the 
relationship between function involving a direct call while light arrow show the 
relationship between two or more. For a complete list of functions available in PsychroLib, 
see the README file in the project’s repository.](psychrolib-relationships.pdf){ width=90% }```

**Comment by dmey** [comment]
Doing a quick search I got to https://github.com/jgm/pandoc/issues/845#issuecomment-369944255 where it may be enough to override the fig placement but not sure if this is something you are looking to change in general.

**Comment by danielskatz** [comment]
I don't think this is really a current issue anymore. A new submission with a similar problem can open a new issue if needed, but in general, it seems like we work out issues like these in the review thread now.
